instagram-app/src/components/SearchBar/SearchBar.css

```diff
@@ -0,0 +1,3 @@
+.SearchBar{
```

Comment:
Try to not leave any empty code in, because another developer might get confused a bit